BuildMasterAutomation/Functions/ConvertFrom-BMOtterScriptExpression.ps1

```diff
@@ -0,0 +1,189 @@
+
+function ConvertFrom-BMOtterScriptExpression
+{
+    <#
+    .SYNOPSIS
+    Converts an Otterscript expression into a PowerShell object.
+
+    .DESCRIPTION
+    The `ConvertFrom-BMOtterScriptExpression` function takes an OtterScript expression as an input and converts it into
+    a PowerShell representation of the object. This function supports converting both `vector` and `map` types into
+    their respective `array` and `hashtable` types in PowerShell.
+
+    .LINK
+    https://docs.inedo.com/docs/executionengine-otterscript-strings-and-literals
+
+    .EXAMPLE
+    "@(1, 2, 3, 4)" | ConvertFrom-BMOtterScriptExpression
+
+    Demonstrates converting an OtterScript vector into a PowerShell array.
+
+    .EXAMPLE
+    "%(hello: world)" | ConvertFrom-BMOtterScriptExpression
+
+    Demonstrates converting an OtterScript map into a PowerShell hashtable.
+
+    .EXAMPLE
+    "%(hello: %(hi: world))" | ConvertFrom-BMOtterScriptExpression
+
+    Demonstrates converting nested OtterScript maps into nested PowerShell hashtables.
+
+    .EXAMPLE
+    "@(1, 2, @(3, 4))" | ConvertFrom-BMOtterScriptExpression
+
+    Demonstrates converting nested OtterScript vectors into nested PowerShell arrays.
+
+    .EXAMPLE
+    "@(1, 2, %(hello: world, hi there: @(5, 6, 7)))" | ConvertFrom-BMOtterScriptExpression
+
+    Demonstrates converting nested OtterScript vectors and maps into nested PowerShell arrays and hashtables.
+    #>
+    [CmdletBinding()]
+    param(
+        # The OtterScript expression to convert to a PowerShell object.
+        [Parameter(Mandatory, ValueFromPipeline)]
+        [String] $Value
+    )
+
+    begin {
+        Set-StrictMode -Version 'Latest'
+        Use-CallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
+
+        function ConvertTo-Int

```

Comment:
Rename this function to `Edit-Output` and update it to both `.Trim()` the passed string and try to parse as an Int.